### PR TITLE
migrate controller yurtappset

### PR DESCRIPTION
**Pull request by huiwq1990**
/kind enhancement
> /kind good-first-issue
> /kind feature
> /kind question
> /kind design
> /sig ai
> /sig iot
> /sig network
> /sig storage

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

File: charts/openyurt/templates/yurt-manager-auto-generated.yaml [hunk 1]
```diff
@@ -878,162 +1528,3 @@ rules:
   - get
   - patch
   - update
----
```
Comment (rambohe-ch) [inline charts/openyurt/templates/yurt-manager-auto-generated.yaml:881]: why do you remove wehbook configurations?

File: go.mod [hunk 2]
```diff
@@ -37,12 +37,15 @@ require (
 	k8s.io/cluster-bootstrap v0.22.3
 	k8s.io/component-base v0.22.3
 	k8s.io/controller-manager v0.22.3
+	k8s.io/klog v1.0.0
```
Comment (rambohe-ch) [inline go.mod:40]: please do not import `k8s.io/klog` package, use `k8s.io/klog/v2` instead.

File: charts/openyurt/templates/yurt-manager-auto-generated.yaml (hunk 1 above)
Comment (huiwq1990) [reply charts/openyurt/templates/yurt-manager-auto-generated.yaml:881]: The webhooks are not deleted, the codes are regenerated.

File: go.mod (hunk 2 above)
Comment (huiwq1990) [reply go.mod]: done

**Review by rambohe-ch — APPROVED** [review]

**Pull request merged**